### PR TITLE
Fixes #1590: Assert more testing parameters to YoutubeScraperTest.java

### LINKED ISSUE #1590: Assert more testing parameters to YoutubeScraperTest.java
- Issue type: Feature request 

### Short description

The goal is to add more testing parameters to YoutubeScraperTest.java file.

### Usecase

This feature will add more testing parameters to increase test coverage.

### Description

The main idea is to increase test coverage by asserting with more parameters.

### PR DESCRIPTION
**Pull request by simsausaurabh**
### Short description
I have:
- [X] There is a corresponding issue for this pull request.
- [X] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [X] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Fixes: #1590

### CODE REVIEW

**Comment by simsausaurabh** [comment]
@singhpratyush please review.

File: test/org/loklak/harvester/YoutubeScraperTest.java [hunk 1]
```diff
@@ -52,13 +53,17 @@ public void parseFromInputStreamTest() throws IOException {
...
+        String postScraper = "youtube";
 
         try {
             //Check Network issue
             BufferedReader br = new BufferedReader(new InputStreamReader((new URL(url)).openStream(), StandardCharsets.UTF_8));
             Post video = ytubeScrape.parseVideo(br, "url", "https://www.youtube.com/watch?v=KVGRN7Z7T1A");
             DAO.log(video.toString());
             assertThat(video.get("html_title").toString(), is("[\"Iggy Azalea - Team (Explicit) - YouTube\"]"));
+            assertEquals(postType, video.get("post_type"));
+            assertEquals(postScraper, video.get("post_scraper"));
```
Comment (singhpratyush) [inline test/org/loklak/harvester/YoutubeScraperTest.java]: Using `assertThat` is preferred here to favor consistency.

Comment (simsausaurabh) [reply test/org/loklak/harvester/YoutubeScraperTest.java]: Are you referring to add the assertions to the other method or to change the ```assertEquals``` to ```assertThat``` or to remove them? @singhpratyush please clarify.

Comment (singhpratyush) [reply test/org/loklak/harvester/YoutubeScraperTest.java]: Do not use `assertEquals` here. Use `assertThat`.

Comment (simsausaurabh) [reply test/org/loklak/harvester/YoutubeScraperTest.java]: Thanks for the clarification, making changes :+1:

**Comment by simsausaurabh** [comment]
@singhpratyush please review.

**Pull request merged**